src/state/messages/index.js

```diff
@@ -1,18 +1,28 @@
-import { handleAction, createAction, handleActions } from 'redux-actions';
+import { createAction, handleActions } from 'redux-actions';
 import { get, last } from 'lodash';
 
 const initialState = {
-  messages: [],
+  messages: [
+    { text: 'hello', timestamp: 3 },
+    { text: 'world', timestamp: 4 }
+  ],
   loading: false,
-  timestamp: 0
+  id: 5,
+  conversetionist: ''
 };
 
+const CLEAR_MESSAGE_BOARD = 'messages: clear-message-board';
 export const FETCH_MESSAGES = 'messages: fetch-messages';
 export const FETCH_MESSAGES_SUCCESS = 'messages: fetch-messages-success';
 const FETCH_MESSAGES_FAILURE = 'messages: fetch-messages-failure';
 
 export const messagesReducer = handleActions({
 
+  [CLEAR_MESSAGE_BOARD]: (state, { payload }) => ({
+    ...state,
+    messages: [{}]
```

Comment:
Do we need this empty object here? Looks strange.

Reply:
You're rught, but I don't know why, but it doesn't work without it.